### ISSUE TITLE
Screenshots generated with incorrect sizes

### ISSUE THREAD

**Issue by staplKody** [FlutterFlow/flutterflow-issues#3962]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When using the screenshot generator, my images are generated a couple pixels off on sizes. 

### Expected Behavior

I would expect the images to be generated with the correct size that the UI indicates. 

### Steps to Reproduce

Preview app
Enter screenshot mode
generate screenshots
check the image size of the generated screenshots

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT8okcjfsMx2rsdY17X5bcAwgikvJlw6aucVlOthbAw0HOrnEooXeOHCUxFVcO26Y0NHBFaj8TgIpuCOkePTU+1cGxCIQ758zKVuewLdXkejb7rUCc/hXEQnO99XJ26E4JqOihZvAONtZVIkwV2he9/IQjHsQpy3DG0nBvyKUZ/Vn2i6HXvWVjh2tmNecCfn

### Visual documentation

You can see here the iphone 6.7 images are generated with dimensions 1288x2794
![image](https://github.com/user-attachments/assets/fa3bf0ec-2af9-4405-b042-715b23624435)

### Environment

```markdown
- FlutterFlow version: FlutterFlow v5.0.2+ 
- Platform: web
- Browser name and version: chrome
- Operating system and version affected:  windows
```

**Comment by Alezanello** [comment]
Hello,

Thank you for bringing this to our attention! I’ve confirmed the issue on our end and will be passing it along to our engineering team for a fix.

We appreciate your patience!

**Comment by jlsimmo** [comment]
Is this fixed yet? I need to publish to iOS but can't because of this,

**Comment by nickybittencourt** [comment]
> Is this fixed yet? I need to publish to iOS but can't because of this,

You can just resize the screenshots generated by FlutterFlow in MSPaint or Preview, they're only a couple of pixels off.

It's annoying to have to do this for a bunch of screenshots but you can at least publish while this issue persists.

**Comment by staplKody** [comment]
I haven't needed to generate new screenshots recently. But it's not terribly annoying to do some resizing via paint.

**Comment by staplKody** [comment]
it appears that this is still not resolved as I just found out.

**Comment by jlsimmo** [comment]
Yes it's annoying... This prevents a major piece of functionality, it's clearly something that must be easy to fix but nothing is being done about it 

I need to republish screen shots because I've infringed someone in the UK's trademark, but I can't because of this bug. 

FlutterFlow ... please fix!

**Comment by staplKody** [comment]
@jlsimmo for now I'm using https://imageresizer.com/bulk-resize/editor to manually resize.

**Comment by sbis04** [comment]
This should be fixed in the next update!

**Comment by staplKody** [comment]
awesome
